.github/workflows/tests.yaml

```diff
@@ -234,14 +236,14 @@ jobs:
         with:
           config: .licenserc.yaml
   required_status_checks:
+    if: "always()"
```

Comment:
I have not tested this, but would enclosing the [`always()` expression](https://docs.github.com/en/actions/learn-github-actions/expressions#always) in quote still work?

Reply:
Yeh I think the quotes are there just so that yaml knows it is a string, they are implied if they are not there

Reply:
I think the double quote is a yaml feature and it will not be there in the evaluate string.